src/py/CMakeLists.txt

```diff
@@ -71,7 +71,6 @@ endif ()
 SET(PYTHON_DEST "${CMAKE_INSTALL_LIBDIR}/python${PYTHON_VERSION}/site-packages" )
 INSTALL(TARGETS _partio DESTINATION ${PYTHON_DEST})
 INSTALL(FILES ${CMAKE_BINARY_DIR}/${outdir}/py/partio.py  DESTINATION ${PYTHON_DEST})
-INSTALL(FILES partio.i DESTINATION share/swig)
```

Comment:
Thanks, this looks great.  Does installing the `partio.i` cause any issues that triggered the removal of the install line?  A better place  for `partio.i` might be `share/partio/swig` instead of `share/swig`.  Any qualms on your end if I reinstate that post-merge in that new location?

The only slightly contentious part about this change is the version bump to require cmake 3.8.2  as a minimum version.  That's okay for me, but hopefully it's not a burden on other users (cmake 3.8.0 was released a year ago).  It's probably worth modernizing nonetheless unless someone chimes in otherwise.

Reply:
1. What requires CMake 3.8.2?
2. `.i` files are pure interface files, and are useless without the rest of the library. This is the first time I see a `.i` file installed, because producing wrappers for other languages (say Ruby for instance) requires adding support for that in your CMake build system.

Reply:
I just saw that.

But why do you want to install the `partio.h` file again? It's pretty similar to installing `.cpp` files, which is useless in itself.

Reply:
We install the foo.i files since it allows downstream users to reuse them.  For example, someone can use:

    // Import the partio swig interface
    %import(module="partio") "partio.i"

in their own swig interface file when writing a new python interface for their library that also returns partio types.  swig will then return the proper object from the partio module when calling into their wrapped library functions.